Packages/vcs/Lib/VTKPlots.py

```diff
@@ -446,8 +447,11 @@ def plot(self,data1,data2,template,gtype,gname,bg,*args,**kargs):
 
     vtk_backend_grid = kargs.get("vtk_backend_grid",None)
     vtk_backend_geo = kargs.get("vtk_backend_geo",None)
-    if gtype in ["boxfill","meshfill","isofill","isoline"]:
-      returned.update(self.plot2D(data1,data2,tpl,gm,vtk_backend_grid=vtk_backend_grid,vtk_backend_geo=vtk_backend_geo))
+
+    pipeline = vcsvtk.createPipeline(gm, self)
+    if pipeline is not None:
+        returned.update(pipeline.plot(data1, data2, tpl, gm,
+                                      vtk_backend_grid, vtk_backend_geo))
     elif gtype in ["3d_scalar", "3d_dual_scalar", "3d_vector"]:
```

Comment:
No real reason - this can be moved later. My priority at this point was to untangle `plot2D`, and I did the vector/1d plots as well since they were easy. At some point all of these should go into pipeline objects.